### DISCUSSION TITLE
Contribution taxonomy

### DISCUSSION THREAD

**Discussion by baharsener** [baharsener/visualEyes#39] [General]
Example things to put in:
Writing functions, writing tests, human-centered design, writing documentation, setting up packaging, setting up CI, code review (maybe more things but at least these) included for everyone in the README

**Comment by baharsener** [comment]
We can also include the things that we would have wanted to include (technologies we would have wanted to use for those things, how we would have tested them, so on)

**Comment by mckenziephagen** [comment]
Other acknowledgements: 
- Project concept: @BrendaQiu 
- Eyelink expertise: @baharsener 
- Bug finder and user testing: @lydiazhanguw 
- Project management: all, but especially @BrendaQiu
